chemicaltoolbox/rdkit/rdkit_get_charge.xml

```diff
@@ -0,0 +1,81 @@
+<tool id="ctb_rdkit_get_charge" name="Formal molecular charge" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
```

Comment:
from a user-point of view, should this be added to https://github.com/bgruening/galaxytools/blob/master/chemicaltoolbox/openbabel/ob_genProp.xml --> even if we cross the rdkit/openbabel boundary here (which is purely technical and does not make much sense to the user maybe?)

Reply:
I guess we could but I'm not so enthusiastic. I think our users are quite well informed about different packages and it should be clear from the tool which one they are using.

Or is your concern generally about creating a lot of tiny tools with very specific functionality?

Reply:
More the latter. More tools means more maintenance and more things that can break.

We could still offer them the information (charge (ob) vs. charge (rdkit)) but having two tools for charge calculation, just with a different implementation seems weird.

But I understand that this is not an important topic. So I'm happy to merge that if you prefer.

Reply:
There is also the rdkit descriptor tool. I can add it to that instead.

> We could still offer them the information (charge (ob) vs. charge (rdkit)) but having two tools for charge calculation, just with a different implementation seems weird.

I don't think charge can be calculated with the openbabel tool?

Reply:
> There is also the rdkit descriptor tool. I can add it to that instead.

Also good! You see we have already so many places. If you concerned about performance, because to many things are calculated we can make those optional. 

> I don't think charge can be calculated with the openbabel tool?

No it can't to my knowledge, it was just an example.